test/tests/api/v2_0/os_install_tests.py

```diff
@@ -46,8 +50,75 @@ def teardown(self):
     def __get_data(self):
         return loads(self.__client.last_response.data)
 
+    def __get_compute_nodes(self):
+        Api().nodes_get_all()
+        nodes = self.__get_data()
+        compute_nodes = []
+        for n in nodes:
+            type = n.get('type')
+            if type == 'compute':
+                compute_nodes.append(n)
+        LOG.info('compute nodes count {0}'.format(len(compute_nodes)))
+        return compute_nodes
+
+    def __wait_for_completion(self, node, graph_name, graph_instance):
```

Comment:
If the `graph_instance` is specified, actually the argument `node` and `graph_name` is redundant. because when the graph instance is created, the node identifier and graph name is known.